e2e/bzlmod/MODULE.bazel

```diff
@@ -5,7 +5,7 @@ module(
 )
 
 bazel_dep(name = "aspect_rules_js", version = "0.0.0")
-bazel_dep(name = "aspect_bazel_lib", version = "2.6.1")
+bazel_dep(name = "aspect_bazel_lib", version = "2.7.0")
```

Comment:
Shouldn't bzlmod take the latest version in the deep tree and therefore none of the e2e changes are necessary?

Reply:
That doesn't mean the MODULE for that test needs the version instrumented though, does it?

Reply:
It has to specify the dependency as a direct dependency same as you would have to in package.json for a direct dep if you're not hoisting. Do you mean that it can specify the dep but with version 0.0.0?